src/openforms/authentication/contrib/digid_oidc/urls.py

```diff
@@ -0,0 +1,21 @@
+from django.conf.urls import url
+
+from mozilla_django_oidc.urls import urlpatterns
+
+from .views import OIDCAuthenticationCallbackView, OIDCAuthenticationRequestView
+
+app_name = "digid_oidc"
+
+
+urlpatterns = [
+    url(
+        r"^callback/$",
+        OIDCAuthenticationCallbackView.as_view(),
+        name="oidc_authentication_callback",
```

Comment:
since we have the `digid_oidc` namespace, I think these can be shortened - drop the `oidc_authentication_` prefix